main.py

```diff
@@ -1,12 +1,207 @@
-    pass
-
+def main(): # what is this main function supposed to do?
+    print_stars()
+    print_logo()
+    print_stars()
 
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
+
+def list_options():
+
+    options = {
+        "1": "Add video",
+        "2": "Update video",
+        "3": "Delete video",
+        "4": "Get info about all videos",
+        "5": "Get info about one video",
+        "6": "Add customer",
+        "7": "Edit customer",
+        "8": "Delete customer",
+        "9": "Get info about one customer",
+        "10": "Get info about all customers",
+        "11": "Check out video to customer",
+        "12": "Check in video from customer",
+        "13": "List all options",
+        "14": "Exit"
+    }
+
+    for choice_num in options:
+        print(f"{choice_num}, {options[choice_num]}")
+
+    return options
+
+def make_choice(options, video):
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print_stars()
+        print(f"What would you like to do?")
+        choice = input("Input an option number: ")
+
+        return choice
+
+def run_cli(play=True):
+
+    # initialize classes
+    video = Video(url=URL)
+    customer = Customer(url=URL)
+    rental = Rental(url=URL)
+
+    # print choices
+    options = list_options()
+
+    while play==True:
+
+        # get input and validate
+        choice = make_choice(options, video)
+
+        # add a video
+        if choice=='1':
+            title=input("Please enter a title: ")
+            release_date=input("Please enter a release date: ")
+            total_inventory=input("How many copies are there total? ")
+            response = video.add_video(title=title, release_date=release_date,
+            total_inventory=total_inventory)
+
+            print(f"New video '{title}' created")
+
+        # update a video
+        elif choice=='2':
+            video_id = input("Which video would you like to update?\n"
+            "Please enter an ID: ")
+            to_integer(video_id)
+
+            title=input("Please enter a new title: ")
+            release_date=input("Please enter a new release date: ")
+            total_inventory=input("How many copies are there total? ")
+
+            response = video.update_video(video_id=video_id, title=title,
+            release_date=release_date, total_inventory=total_inventory)
+
+            print_stars()
+            print(f"Updated video id {video_id}")
+
+        # delete a video
+        elif choice=='3':
+            video_id = input("Which video would you like to delete?\n"
+            "Please enter an ID: ")
+            to_integer(video_id)
+
+            video.delete_video(video_id)
+
+            print(f"Video {video_id} has been deleted")
+
+        # get info about all videos
+        elif choice=='4':
+            print_stars()
+            for video in video.list_videos():
+                print(video)
```

Comment:
This for loop is causing subtle errors.

```suggestion
            video_list = video.list_videos()
            for vid in video_list:
              print(vid)
```